datasets/eu/esma_saris/crawler.py

```diff
@@ -31,31 +31,26 @@ def crawl(context: Context) -> None:
             entity.add("name", row.pop("instrumentFullName", isin))
 
             sanction = h.make_sanction(context, entity)
-            sanction.id = context.make_id(row.pop("id"))
             sanction.add("program", "ESMA")
             sanction.add("provisions", row.pop("actionType"))
             sanction.add("reason", row.pop("reasonsForTheAction"))
             sanction.add("description", row.pop("comments"))
             sanction.add("startDate", row.pop("effectiveFrom"))
             sanction.add("country", row.pop("memberStateOfNotifyingCA"))
             sanction.set("authority", row.pop("notifyingCA"))
-            target = False
-            if row.get("effectiveTo") is not None:
-                sanction.add("endDate", row.pop("effectiveTo"))
-            else:
-                entity.add("topic", "sanction")
-                target = True
-            context.emit(sanction, target=target)
-
-            if row.get("issuer") is not None:
+            sanction.add("endDate", row.pop("effectiveTo"))
```

Comment:
I had some logic here that only added the `sanction` topic if the sanction hadn't ended, but looking back I guess that's wrong? The topic will always be `sanction` for this dataset, but if it has an end-date you'll know that the sanction no longer is enforced.

Reply:
OK, I restored it :)